backend/Shared/HitMeApp.Shared.DDD/EntityId.cs

```diff
@@ -3,16 +3,16 @@
 
 namespace HitMeApp.Shared.DDD
 {
-    public class AggregateId : IEquatable<AggregateId>
+    public abstract class EntityId : IEquatable<EntityId>
     {
         public Guid Value { get; protected init; }
```

Comment:
This can be get-only if we're assigning it from constructor

Reply:
I left it on purpose in case of some ORM mapping problems when we introduce DB.
Some ORMs required at least protected set to work correctly which was a shame, but maybe it was fixed in .NET 5.
I'll test it after DB introduction.

Reply:
Oh I see. Allright then.